idunn/places/pj_poi.py

```diff
@@ -388,6 +394,16 @@ def get_images_urls(self):
     def get_source(self):
         return PoiSource.PAGESJAUNES
 
+    def get_source_url(self):
+        return next(
+            (
+                ins.urls.merchant_url
```

Comment:
We may want to avoid exposing users to some stats?

```suggestion
                resolve_url(ins.urls.merchant_url)
```

They should end up on pagesjaunes.fr anyway so there may not be much more information that could be fetched. (Maybe if we want to be extra safe we could check that the link actually redirects to pagejaunes.fr?)